src/main/java/racing/domain/car/Car.java

```diff
@@ -0,0 +1,55 @@
+package racing.domain.car;
+
+import racing.domain.RandomNumberPicker;
+import racing.domain.race.NumberPicker;
+
+public class Car {
+    private static final int LEAST_CONDITION = 4;
+    private static final int MAX_NAME_LENGTH = 5;
+    private static final int ONE_STEP = 1;
+    private final String name;
+    private final Position position;
+    private NumberPicker numberPicker = new RandomNumberPicker();
```

Comment:
```suggestion
    private static final int ONE_STEP = 1;
    
    private final String name;
    private final Position position;
    
    private NumberPicker numberPicker = new RandomNumberPicker();
```

지금도 좋지만 개행이 한 칸씩 추가되면 더 좋을 것 같아요 ㅎㅎ